### PR TITLE
speed up client tests

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/presenter/presenter-mock.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+export const presenter = {
+  providers: {},
+  state: {
```
Comment (matthopson) [inline web-client/src/presenter/presenter-mock.js:3]: Should we further break down state? Like a `baseState` with all these keys, that we can pull in here from `state.js`?

File: shared/src/business/test/createTestApplicationContext.js [hunk 2]
```diff
@@ -248,17 +246,12 @@ const createTestApplicationContext = ({ user } = {}) => {
...
-        webClientApplicationContext.getChiefJudgeNameForSigning,
-      ),
+    getChiefJudgeNameForSigning: jest.fn(),
     getChromiumBrowser: jest.fn(),
     getClerkOfCourtNameForSigning: jest.fn(),
     getCognito: appContextProxy(),
     getConstants: jest.fn().mockReturnValue({
-      ...webClientApplicationContext.getConstants(),
-      ...webApiApplicationContext.getConstants(),
+      ...getConstants(),
```
Comment (kkoskelin) [inline shared/src/business/test/createTestApplicationContext.js:254]: Unless there is a test somewhere checking on how/whether `getConstants` has been called, you can just drop the real reference directly in here.  That goes for anything in here, tbh.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/presenter-mock.js (hunk 1 above)
Comment (kkoskelin) [reply web-client/src/presenter/presenter-mock.js:3]: I'm not even convinced that we need hardly anything in this file. state could be an empty object and I think they'd still pass.

**Review by sutt0n — APPROVED** [review]

**Pull request merged**